src/main/java/gil/server/Routes.java

```diff
@@ -1,11 +1,13 @@
 package gil.server;
 
 public class Routes {
-    private final String ROUTE_NOT_FOUND = "route-not-found";
+    public static final String ROUTE_NOT_FOUND = "route-not-found";
+    public static final String STATIC_FILE_OPTIONS = "static-file-options";
 
     public void addRoutes(Router router) {
         router.get(ROUTE_NOT_FOUND, RouteNotFoundController.get);
         router.get("/", RootEndpointController.get);
         router.get("/api/parameters", ParametersController.get);
+        router.options(STATIC_FILE_OPTIONS, StaticFileOptionsController.options);
```

Comment:
As I mentioned during the IPM, it'd be good to not do this. just a reminder!

Reply:
Thanks for reminding me! 👍